cmd/run.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"github.com/astronomer/astro-cli/cmd/utils"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dagID     string
+	startDate string
+	taskLogs  bool
+)
+
+func newRunCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "run DAG-ID",
+		Short: "Run a local DAG withpython by running its tasks sequentially",
+		Long:  "Run a local DAG by running its tasks sequentially. This command will spin up a docker airflow environment and execute your DAG code",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
```suggestion
		Args:  cobra.ExactArgs(1),
```
`ExactArgs` will also ensure case when someone fails to pass the dag-id as well as case when more than one argument is passed